wpt-metadata/chromedriver/headful/webdriver/tests/bidi/network/response_completed/response_completed_status.py.ini

```diff
@@ -0,0 +1,2 @@
+[response_completed_status.py]
```

Comment:
We should be consistent, either `disabled` or `implementation-status: backlog`.

Reply:
This is a copy-paste from headless. Do you think cleaning things up should be in this PR? Should it also include changes in headless metadata?

Reply:
Please resolve the conversation if you find it's out of scope of this PR.

Reply:
OK, this is a fair argument not to address it in this PR.